scripts/add_existing_baseyear.py

```diff
@@ -492,6 +489,7 @@ def add_chp_plants(n, grouping_years, costs, baseyear, clustermaps):
     ppl = pd.read_csv(snakemake.input.powerplants, index_col=0)
 
     if snakemake.input.get("custom_powerplants"):
+        ppl = ppl.query("~(Set == 'CHP' and Country == 'DE')")
```

Comment:
This line implicitly assumes that the custom_powerplants supersede the data for CHPs in DE.

However, if the data in custom_powerplants changes, this might no longer be true. I would like to see a more explicit solution that applies this filter only, when the mastr_chps are read in.

This is important since add_existing is a pypsa-eur script and soon we will merge ariadne2 completly into the master branch.